draft-ietf-core-cf-reg-update.md

```diff
@@ -214,6 +216,11 @@ If a temporary registration (in any range) refers to a provisional media type th
 Note that in the 10000-64999 range the abandonment of a document requesting a Content-Format ID does not cause an entry to be removed.
 That is because the required registration procedure for this range does not require completion of any standards process, nor does it require a registering document.
 
+{:#expert-review-7120-exemptions}
+Temporary registrations within the 0-255 range are exempt from the formal renewal process outlined in {{-iana-early}}.
+Specifically, IANA will not monitor the removal of registrations in this range.
+Instead, the Designated Expert directs IANA to carry out this task.
```

Comment:
If here too the intent is to refer to "someone with the DE role", then it should be:

> Instead, Designated Experts direct IANA to carry out this task.